src/export/sketch-exporter.ts

```diff
@@ -21,6 +28,74 @@ export class SketchExporter {
 			getRectangleName: () => 'background',
 			skipSystemFonts: true
 		});
-		FileExtraUtils.writeFileSync(path, JSON.stringify(sketchPage.toJSON(), null, '\t'));
+
+		return JSON.stringify(sketchPage.toJSON(), null, '\t');
 	}
+
+	// tslint:disable-next-line promise-function-async
+	public createExport(webview: WebviewTag): Promise<ExportResult> {
+		return new Promise((resolve, reject) => {
+			// tslint:disable-next-line no-any
+			ipcRenderer.on('export-as-sketch-done', (_: Event, payload: any) => {
```

Comment:
I guess you should either listen globally once, or safely unlisten afterwards in any way. Currently, more and more SketchExporter instances collect.